projects/ngrx-auto-entity/src/lib/selectors/selectors.spec.ts

```diff
@@ -85,107 +76,107 @@ describe('mapToCustomSortedEntityArray()', () => {
 describe('mapToLoadedAt()', () => {
   it('should return null if state falsy', () => {
     const ts = mapToLoadedAt(null);
-    expect(ts).toBeNull();
+    expect(ts).toBeUndefined();
   });
 
   it('should return null if loadedAt state is falsy', () => {
-    const ts = mapToLoadedAt({ entities: {}, ids: [] });
-    expect(ts).toBeNull();
+    const ts = mapToLoadedAt({});
+    expect(ts).toBeUndefined();
   });
 
   it('should return Date', () => {
     const now = Date.now();
-    const ts = mapToLoadedAt({ entities: {}, ids: [], loadedAt: now });
+    const ts = mapToLoadedAt({ loadedAt: now });
     expect(ts).toStrictEqual(new Date(now));
   });
 });
 
 describe('mapToSavedAt()', () => {
   it('should return null if state falsy', () => {
     const ts = mapToSavedAt(null);
-    expect(ts).toBeNull();
+    expect(ts).toBeUndefined();
   });
 
   it('should return null if savedAt state is falsy', () => {
-    const ts = mapToSavedAt({ entities: {}, ids: [] });
-    expect(ts).toBeNull();
+    const ts = mapToSavedAt({});
+    expect(ts).toBeUndefined();
   });
 
   it('should return Date', () => {
     const now = Date.now();
-    const ts = mapToSavedAt({ entities: {}, ids: [], savedAt: now });
+    const ts = mapToSavedAt({ savedAt: now });
     expect(ts).toStrictEqual(new Date(now));
   });
 });
 
 describe('mapToCreatedAt()', () => {
   it('should return null if state falsy', () => {
```

Comment:
```suggestion
  it('should return undefined if state falsy', () => {
```
Not a big deal, but should we go ahead and change all of these since it's undefined instead of null now?